src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -635,7 +440,8 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
       );
     }
 
-    if (!isLegacyFile && status === 'success') {
+    // Import summary for completed import
```

Comment:
Not sure if the code comment is needed here so I added one anyway. We only had a comment for handling legacy imports and that's gone now.